### PR TITLE
qsreplace fixes #3098

### LINKED ISSUE #3098: qsreplace
qsreplace
https://github.com/tomnomnom/qsreplace

qsreplace
Accept URLs on stdin, replace all query string values with a user-supplied value, only output each combination of query string parameters once per host and path.

Author: tomnomnom

### PR DESCRIPTION
**Pull request by dualfade**
please review --
Thanks.

### CODE REVIEW

File: packages/qsreplace/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+pkgname=qsreplace
+pkgver=3.0b053d2
+pkgrel=1
+pkgdesc='Accept URLs on stdin, replace all query string values with a user-supplied value, only output each combination of query string parameters once per host and path.'
+arch=('x86_64' 'aarch64')
+groups=('blackarch' 'blackarch-misc')
+url='https://github.com/tomnomnom/qsreplace'
+license=('MIT')
+depends=()
```
Comment (Edu4rdSHL) [inline packages/qsreplace/PKGBUILD:12]: Remove empty array.

File: packages/qsreplace/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+makedepends=('git' 'go')
+source=("git+https://github.com/tomnomnom/$pkgname.git")
+sha512sums=('SKIP')
+_url="github.com/tomnomnom/qsreplace"
```
Comment (Edu4rdSHL) [inline packages/qsreplace/PKGBUILD:16]:
```suggestion

```

File: packages/qsreplace/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+pkgver() {
+  cd $pkgname
+
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+}
+
+build() {
+  cd $pkgname
+
+  GOPATH="$srcdir/$pkgname" go mod init $_url
```
Comment (Edu4rdSHL) [inline packages/qsreplace/PKGBUILD:27]:
```suggestion
  GOPATH="$srcdir" go mod init go.mod
```

File: packages/qsreplace/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  GOPATH="$srcdir" go mod tidy
```
Comment (Edu4rdSHL) [inline packages/qsreplace/PKGBUILD:28]:
```suggestion
  GOPATH="$srcdir" go mod tidy
  GOPATH="$srcdir" go mod download
```

File: packages/qsreplace/PKGBUILD [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
```
Comment (Edu4rdSHL) [inline packages/qsreplace/PKGBUILD:29]:
```suggestion
```

**Review by Edu4rdSHL — CHANGES_REQUESTED** [review]

File: packages/qsreplace/PKGBUILD (hunk 1 above)
Comment (dualfade) [reply packages/qsreplace/PKGBUILD:12]: removed

Comment (Edu4rdSHL) [reply packages/qsreplace/PKGBUILD:12]: Please check/apply the new suggestions too.

**Comment by dualfade** [comment]
Thanks! @Edu4rdSHL

**Comment by Edu4rdSHL** [comment]
Thank you a lot!

**Pull request merged**